Makefile

```diff
@@ -40,13 +40,17 @@ ENV_DOCKER             ?= docker
 ENV_DOCKER_COMPOSE     ?= docker-compose --project-directory $(CURDIR) -p $(project_name) -f $(project_compose_ci)
 ENV_NGINX              ?= $(ENV_NGINX_EXEC) -p $(CURDIR) -c $(CURDIR)/conf/nginx.conf
 ENV_NGINX_EXEC         := $(shell command -v openresty 2>/dev/null || command -v nginx 2>/dev/null)
-ENV_OPENSSL_PREFIX     ?= $(addprefix $(ENV_NGINX_PREFIX), openssl3)
+ENV_OPENSSL_PREFIX     ?= /usr/local/openresty/openssl3
```

Comment:
why change this?

Reply:
By default, `ENV_ NGINX_ PREFIX` is empty.
Only when it is set, or when openssl3 dir is empty, it will take effect in the future